chia/full_node/weight_proof.py

```diff
@@ -618,6 +618,9 @@ def get_fork_point(self, received_summaries: List[SubEpochSummary]) -> uint32:
         for idx, summary_height in enumerate(ses_heights):
             log.debug(f"check summary {idx} height {summary_height}")
             local_ses = self.blockchain.get_ses(summary_height)
+            if idx == len(received_summaries) - 1:
+                # break if end of wp summaries
```

Comment:
a bit of a nit, but this comment seems pretty pointless. I think it would be better to have a comment explaining why we're done with the loop in this case

Reply:
updated